vcf-dumper/README.md

```diff
@@ -0,0 +1,122 @@
+# European Variation Archive (EVA) VCF dumper
+
+The VCF dumper is a web service that supports some queries from the [EVA core REST Webservices API](https://github.com/EBIvariation/eva-ws/wiki#variants "EVA WS Variants endpoints") and writes the output in [VCF format](https://samtools.github.io/hts-specs/VCFv4.3.pdf "VCF format specification")
```

Comment:
If there is already the paragraph text I don't see a big point in adding a popup...